exercises/luhn/luhn.py

```diff
@@ -1,3 +1,3 @@
 class Luhn(object):
-    def __init__(self):
+    def __init__(self, string):
```

Comment:
This is too general. 
Look at https://github.com/exercism/python/tree/master/exercises/luhn it requires not just a string, it requires string of numbers.
Could you propose a better name?

Reply:
it does not requires a string of numbers, it validates a string of numbers.
From the luhn/README.md: 
```
The Luhn algorithm is a simple checksum formula used to validate a variety of identification numbers,
```

Reply:
@jnaulty I see, maybe I was not clear enough in my comment. 
My vision is to provide good parameter names to the learner, as clearer as possible.
What about `string_with_numbers`?

Concerning test cases, there is a special repository for defining exercises - https://github.com/exercism/problem-specifications/. For this exercise it is https://github.com/exercism/problem-specifications/blob/master/exercises/luhn/canonical-data.json. Not sure if it's a good idea to provide a test case for non string input, even though I personally always add this kind of verifications